app/views/browse/pod.html

```diff
@@ -124,19 +124,60 @@
+                      They will remain connected until you navigate away from this page.
                     </div>
-                    <div class="pod-container-terminal"
-                      ng-repeat="container in containers | orderBy:'name' track by container.name"
-                      ng-if="container.state.running">
-                      <h3 ng-if="containers.length > 1">{{container.name}}</h3>
-                      <kubernetes-container-terminal pod="pod" container="container.name"
-                        prevent="!terminalTabWasSelected">
-                      </kubernetes-container-terminal>
+
+                    <div>
+                      <ui-select
+                        ng-model="selected"
+                        on-select="onTerminalSelectChange(selected)"
+                        class="mar-left-none pad-left-none col-xs-3">
+                        <ui-select-match
+                          placeholder="Select or search">
+                            <span
+                              ng-class="{'btn-success' : ($select.selected.value.isUsed && ($select.selected.value.status === 'connected')),
+                                         'btn-danger' : ($select.selected.value.status === 'disconnected'),
+                                         'mar-left-lg' : !$select.selected.value.isUsed}">
+                                <i class="fa fa-terminal" ng-show="$select.selected.value.isUsed"></i>
+                            </span>
+                            <span class="mar-left-sm" ng-bind-html="$select.selected.value.name | highlight: $select.search"></span>
+                        </ui-select-match>
+                        <ui-select-choices
+                          class="col-xs-3"
+                          repeat="(key, term) in containerTerminals">
+                          <div>
+                            <span
+                              ng-class="{'btn-success' : (term.value.isUsed && (term.value.status === 'connected')),
+                                         'btn-danger' : (term.value.status === 'disconnected'),
+                                         'mar-left-lg' : !term.value.isUsed}">
+                                <i class="fa fa-terminal" ng-show="term.value.isUsed"></i>
+                            </span>
+                            <!--span class="mar-left-lg" ng-show="!term.value.isUsed && !term.value.died"></span>
+                            <span class="pficon-error-circle-o" ng-show="term.value.died"></span-->
+                            <span class="mar-left-sm" ng-bind-html="term.value.name | highlight: $select.search"></span>
+                          </div>
+                        </ui-select-choices>
+                      </ui-select>
+
+                      <br><br> <!-- Padding/margin didn't give me the line separation I wanted. Recommendations? -->
```

Comment:
Just for some spacing, right?  styles/_spacers.less defines some utillity classes that you can use, such as `.pad-bottom-md`